modules/subscriptions/views.php

```diff
@@ -312,8 +320,16 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 						echo wpautop( $subscribe_logged_in );
 					}
 					if ( $show_subscribers_total && $subscribers_total ) {
-						/* translators: %s: number of folks following the blog */
-						echo wpautop( sprintf( _n( 'Join %s other follower', 'Join %s other followers', $subscribers_total ), number_format_i18n( $subscribers_total ) ) );
+						?>
+						<div class="subscribe-count">
```

Comment:
Should we prefix this with `jetpack-` to not accidentally collide with any other plugin?

Reply:
Yes, will add.